radicle-httpd/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "radicle-httpd"
+license = "MIT OR Apache-2.0"
+version = "0.1.0"
+authors = ["Alexis Sellier <alexis@radicle.xyz>"]
+edition = "2021"
+default-run = "radicle-httpd"
+build = "../build.rs"
+
+[features]
+default = []
+logfmt = [
+  "tracing-logfmt",
+  "tracing-subscriber/env-filter"
+]
+
+[dependencies]
+anyhow = { version = "1" }
+flate2 = { version = "1" }
+lexopt = { version = "0.2.1" }
+thiserror = { version = "1" }
+tokio = { version = "1.21", default-features = false, features = ["macros", "rt-multi-thread"] }
+tracing = { version = "0.1.37", default-features = false, features = ["std", "log"] }
+tracing-subscriber = { version = "0.3", default-features = false, features = ["std", "ansi", "fmt"] }
+tracing-logfmt = { version = "0.2", optional = true }
+axum = { version = "0.5.16", default-features = false }
+axum-server = { version = "0.4.2", default-features = false }
+hyper = { version = "0.14.17", default-features = false }
+tower-http = { version = "0.3.4", default-features = false, features = ["trace"] }
+
+[dependencies.radicle]
+path = "../radicle"
+version = "0.2.0"
```

Comment:
Side note, and not directly related to these changes.

Why specify the patch version in the Cargo file?  Specially for versions with patch level '0'.  [Cargo's spec](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio) define version's with a patch level as lower bounds.  It will resolve '0.2.0' to the highest patch level, maybe '0.2.9' for instance.

Specifying just the major and minor version number would remove redundant information, and may be best to include it only when necessary.  For example using 'axum-server' version '0.4.*' (or 0.4) and leaving Cargo to choose the best patch version.

Reply:
Hmm, in the above case it's not necessary, but I thought that eg. `0.2.1` would select `0.2.1` and above, vs. `0.2` which might select `0.2.0`? Maybe I'm wrong, I find semver really confusing.

Reply:
> ... but I thought that eg. `0.2.1` would select `0.2.1` and above, vs. `0.2` which might select `0.2.0`? ...

That's correct. Though since cargo selects the best possible patch version, why include them unless it affects builds?  We'll avoid maintaining minor patch versions in the Cargo.toml.

Reply:
Ah gotcha, yeah it can be omitted; but it might be useful to know if it's not selecting the latest..